templates/common/config/00-config.conf

```diff
@@ -18,7 +18,6 @@ max_logfile_size_mb=50
 log_rotation_type=size
 log_file = {{ .LogFile }}
 {{ end }}
-enabled_backends=default_backend:file
```

Comment:
@konan-abhi wondering if we should also remove the following:

```
[glance_store]
default_backend=default_backend
```
on L46-L47

Reply:
If we are injecting it at runtime then yes, we can remove it as well.

Reply:
ack thank you, updating the patch

Reply:
Done